appserver/payara-appserver-modules/microprofile/metrics/src/main/java/fish/payara/microprofile/metrics/MetricsService.java

```diff
@@ -126,6 +140,80 @@ public void init() {
         }
     }
 
+    @Override
+    public void collect(MonitoringDataCollector rootCollector) {
+        MonitoringDataCollector metricsCollector = rootCollector.in("metric");
```

Comment:
Minor but may you name it: `metrics`

Reply:
So far I did use singular for the namespace and type tags. Overall this reads better but this is all up to discussion. I should just end up consistent.